etl/steps/data/garden/energy/2023-07-10/photovoltaic_cost_and_capacity.py

```diff
@@ -0,0 +1,167 @@
+"""Combine data from Nemet (2009), Farmer & Lafond (2016) and IRENA on photovoltaic cost and capacity.
+
+Data content:
+* Nemet (2009) provides cumulative capacity data between 1975 and 2003.
+* Nemet (2009) provides cost data between 1975 and 2003.
+* IRENA provides cumulative capacity data between 2000 and 2021.
+* IRENA provides cost data between 2010 and 2021.
+* Farmer & Lafond (2016) provide cost data between 1980 and 2013.
+
+For each informed year, we need to combine these sources with the following two constraints:
+* Having data from the most recent source.
+* Avoid (as much as possible) having cost and capacity data on a given year from different sources.
+
+Therefore, for capacity data, we use Nemet (2009) between 1975 and 2003, and IRENA between 2004 and 2021.
+For cost data, we use Nemet (2009) between 1975 and 2003, Farmer & Lafond (2016) between 2004 and 2009, and IRENA between 2010 and 2021.
+
+"""
+
+import owid.catalog.processing as pr
+from owid.catalog import Dataset, Table
+from owid.catalog.tables import (
+    get_unique_licenses_from_tables,
+    get_unique_sources_from_tables,
+)
+from owid.datautils.dataframes import combine_two_overlapping_dataframes
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current data step.
+paths = PathFinder(__file__)
+
+# Conversion factors.
+# Convert 2004 USD to 2021 USD.
+USD2004_TO_USD2021 = 1.42
```

Comment:
I don't understand your question. I can't access [the Nemet 2009 paper](https://www.sciencedirect.com/science/article/abs/pii/S0301421508005910), but the axis label of [the chart](https://pcdb.santafe.edu/graph.php?curve=158) of the Performance Curve Database shows "2004 USD/Watt", so  I understand that it means constant 2004 USD.
And [the Farmer & Lafond (2016) paper](https://www.sciencedirect.com/science/article/pii/S0048733315001699) talks about "2013 dollars".

Reply:
I mean that those values need to be adjusted for inflation to make that correction, It's very likely they are, but I don't know.

Reply:
If they explicitly say 2004$ or 2013$, while showing data for many different years, it would be odd if they were not correcting for inflation, wouldn't it?

Reply:
That's right. Now I saw one of the papers. So it should be fine. The other consideration is to use a GDP deflator or CPI. I guess in this case it might be right to use a GDP deflator because this is not part of the consumption basket